sdks/python/apache_beam/dataframe/__init__.py

```diff
@@ -14,4 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Beam DataFrame API
+
+- For high-level documentation see
+  https://beam.apache.org/documentation/dsls/dataframes/overview/
+- :mod:`apache_beam.dataframe.io`: DataFrame I/Os
+- :mod:`apache_beam.dataframe.frames`: DataFrame operations
+- :mod:`apache_beam.dataframe.convert`: Conversion between
+  :class:`~apache_beam.pvalue.PCollection` and
+  :class:`~apache_beam.dataframe.frames.DeferredDataFrame`.
+- :mod:`apache_beam.dataframe.transforms`: Embed DataFrame operations in a
+  Beam pipeline.
```

Comment:
"Embed DataFrame operations in a Beam pipeline." -> "DataFrame operations for use in a Beam pipeline."

Reply:
Done. Thank you!